racket/collects/pkg/private/new.rkt

```diff
@@ -222,9 +222,8 @@ EOS
 ;; To view documentation:
 ;;   $ raco docs <<name>>
 ;;
-;; For your convenience, we have included a LICENSE.txt file, which links to
-;; the GNU Lesser General Public License.
```

Comment:
Is this bit of information user facing somewhere? Perhaps `raco` output? or is it only a source code comment?

Reply:
Ignore this comment, I just noticed in an earlier commit that this is user facing. It's weird though that the comment is down below. Is there any reason for it being located where it is, or is it misplaced?

Reply:
for other reviewers - this section was also changed to refer to MIT and APACHE licenses,

``` 
;; To view documentation:
;;   $ raco docs <<name>>
;;
;; For your convenience, we have included LICENSE-MIT and LICENSE-APACHE files.
;; If you would prefer to use a different license, replace those files with the
;; desired license.
```